src/Compilers/VisualBasic/Test/Emit/ErrorHandling.vb

```diff
@@ -1037,6 +1037,51 @@ End]]>)
 ]]>)
         End Sub
 
+        <Fact>
+        Public Sub SynthesizingDefaultConstructorsWithMsCorLibMissing_Nothing()
+            Dim comp = CompilationUtils.CreateCompilation("
+Namespace System
+    Public Class Object
+    End Class
+End Namespace
+Class Test
+End Class
+")
+
+            CompilationUtils.AssertTheseDiagnostics(comp, <expected>
+BC30002: Type 'System.Void' is not defined.
+    Public Class Object
+    ~~~~~~~~~~~~~~~~~~~~
+BC30183: Keyword is not valid as an identifier.
```

Comment:
Will add a couple more tests (one without void, and one with object but no default constructor).

---
In reply to: [115869783](https://github.com/dotnet/roslyn/pull/18715#discussion_r115869783) [](ancestors = 115869783,115615119,115349973)